cli/flags/flags.go

```diff
@@ -12,11 +12,13 @@ const (
 	threshold                = "threshold"
 	withdrawAddress          = "withdrawAddress"
 	operatorIDs              = "operatorIDs"
-	operatorsInfo            = "operatorsInfoPath"
+	operatorsInfoFile        = "operatorsInfoFile"
```

Comment:
this name is a bit misleading, lets make it `operatorsInfo`, make sure to add in the help section we're expecting a json format string.